debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -322,13 +328,17 @@ const djdt = {
             return value;
         },
     },
+    get_debug_element: function () {
+        return document.getElementById("djDebug");
+    },
 };
 window.djdt = {
     show_toolbar: djdt.show_toolbar,
     hide_toolbar: djdt.hide_toolbar,
     init: djdt.init,
     close: djdt.hide_one_level,
     cookie: djdt.cookie,
+    get_debug_element: djdt.get_debug_element,
```

Comment:
I'm a bit sad that we're using a mix of snake_case and camelCase in this file; camelCase seems to be more idiomatic when writing javascript since almost all of the JS runtime itself uses camelCase (especially the DOM) but consistency is more important in the end.

The more important question is maybe if `get_debug_element` really has to be exposed here? Can't the method "just" be an internal helper?

Reply:
You're right. It can be internal. I'll switch it to camelCase. And then have another commit changing everything else to camelCase so we can stay consistent moving forward.